.gitpod.Dockerfile

```diff
@@ -5,5 +5,12 @@ ENV RETRIGGER=1
 ENV BUILDKIT_VERSION=0.9.3
 ENV BUILDKIT_FILENAME=buildkit-v${BUILDKIT_VERSION}.linux-amd64.tar.gz
 
-# Install custom tools, runtime, etc.
-RUN sudo su -c "cd /usr; curl -L https://github.com/moby/buildkit/releases/download/v${BUILDKIT_VERSION}/${BUILDKIT_FILENAME} | tar xvz"
+# Install dazzle and buildkit
+RUN cd /usr \
+    && curl -sSL https://github.com/moby/buildkit/releases/download/v${BUILDKIT_VERSION}/${BUILDKIT_FILENAME} | sudo tar -xvz \
+    && curl -sSL https://github.com/gitpod-io/dazzle/releases/download/v0.1.6/dazzle_0.1.6_Linux_x86_64.tar.gz | sudo tar -xvz dazzle
+
+# Install pre-commit dependencies
+RUN sudo curl -sSL https://github.com/mvdan/sh/releases/download/v3.4.2/shfmt_v3.4.2_linux_amd64 -o /usr/bin/shfmt \
```

Comment:
Can we club the two run commands together?

Reply:
Yeah. But here's a noob question: do fewer layers mean faster build/pull?

Reply:
@axonasif each `RUN` is a new layer: 
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
Yeah, it impacts the image size and build time, right?
I like to chain all commands in one RUN thinking that it'll make things faster.